routes/chat.js

```diff
@@ -18,7 +18,7 @@ const {
   searchGif,
   listFeaturedGif,
   registerShareGif,
-  initChatFromPost
+  initChatFromPost,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements for `groupAddMembers` and `removeGroupMember` have been added, but the corresponding comma separating them from the previous import `initChatFromPost` is missing. This will cause a syntax error.
```diff
-  initChatFromPost
+  initChatFromPost,
   groupAddMembers,
   removeGroupMember
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->